repair/row_level.hh

```diff
@@ -214,6 +214,7 @@ public:
             shard_config master_node_shard_config,
             table_schema_version schema_version,
             streaming::stream_reason reason,
+            std::optional<gc_clock::time_point> compaction_time,
```

Comment:
Is there nothing in idl for this? we added verb support a long time ago.

Please sanity test manually mixed version repair in both directions.

Reply:
Not sure what you mean. This is not the verb definition itself. But I also amended that one.

Reply:
We have things like

```
verb [[with_client_info, cancellable]] direct_fd_ping (raft::server_id dst_id) -> service::direct_fd_ping_reply;
```

I thought we converted all verbs, but maybe not.

Reply:
Looks like repair was not converted. @asias ?

Reply:
Not all verbs are converted.